docs/Config.md

```diff
@@ -61,6 +61,7 @@ gui:
   showIcons: false
   commandLogSize: 8
   splitDiff: 'auto' # one of 'auto' | 'always'
+  skipRewordInEditorPrompt: false # for skipping the confirmation before launching the reword editor
```

Comment:
```suggestion
  skipRewordInEditorWarning: false # for skipping the confirmation before launching the reword editor
```
This is more consistent with existing config keys

Reply:
OK, makes sense. Here's a fixup: 73316fac670e.

(A fixup doesn't makes a huge amount of sense for this little PR, also because all the formatting changes, so I'm going to squash it right away.)

Reply:
Argh, and the fixup wasn't even complete because I forgot to save `Config.md`. Sorry for the mess. But the current state should be good now.